src/sentry/static/sentry/app/components/idBadge/projectBadge.tsx

```diff
@@ -1,27 +1,72 @@
 import React from 'react';
+import styled from '@emotion/styled';
 
 import BadgeDisplayName from 'app/components/idBadge/badgeDisplayName';
 import BaseBadge from 'app/components/idBadge/baseBadge';
+import Link from 'app/components/links/link';
+import {Organization} from 'app/types';
+import withOrganization from 'app/utils/withOrganization';
 
 type BaseBadgeProps = React.ComponentProps<typeof BaseBadge>;
 type Project = NonNullable<BaseBadgeProps['project']>;
 
 type Props = Partial<Omit<BaseBadgeProps, 'project' | 'organization' | 'team'>> & {
   project: Project;
+  organization?: Organization;
   /**
    * If true, will use default max-width, or specify one as a string
    */
   hideOverflow?: boolean | string;
+  /**
+   * If true, this component will not be a link to project details page
+   */
+  disableLink?: boolean;
```

Comment:
should we call this prop `withLink` or `isLink` since `disableLink` is set to false by default and if false the UI does not render the Link component?